src/modules/app/app.controller.ts

```diff
@@ -117,6 +119,68 @@ export class AppController {
     }
   }
 
+  @Post("/validate/file")
+  @ApiHeader({
+    name: "x-oapi-url",
+    description: "Pass the JSON.",
+    allowEmptyValue: false,
+  })
```

Comment:
is this api taking headers "x-oapi-url"?

Reply:
yes, it is taking and we are also passing that from FE.

Reply:
it is not required